public/css/application.css

```diff
@@ -0,0 +1,18 @@
+.container {
+  margin: 20px auto;
+  width: 600px;
+  padding: 0px 20px 20px 20px;
+  border: 1px dotted black;
+}
+th {
```

Comment:
This is too broad of CSS declaration. This changes all `th` elements. Which you dont want. Make this CSS selector more specific.